alternator/consumed_capacity.hh

```diff
@@ -41,6 +41,12 @@ public:
      * \brief get_half_units calculate the half units from the total bytes based on the type of the request
      */
     virtual uint64_t get_half_units() const noexcept = 0;
+    uint64_t& get_total_bytes() {
```

Comment:
On first reading I didn't realize this isn't a traditional "getter" that returns a value - rather it returns a reference. Not even a const reference.
In this case, I propose that:
1. Call it something different, e.g., ref_total_bytes()
2. Get rid of the setter, since you can just do ref_total_bytes() = 3 to set the value.

On second thought, why even have a function at all? Just make a member "total_bytes" (without underscore) public and allow it to be used directly. What's the downside of that?

Reply:
Originally I thought about using the second option, but it looks confusing so I added an extra setter, I'll just make it public